fjord/feedback/tests/test_views.py

```diff
@@ -102,6 +103,23 @@ def test_valid_sad(self):
         eq_(u'Firefox', feedback.product)
         eq_(u'14.0.1', feedback.version)
 
+    @with_waffle('thankyou', True)
+    def test_thank_you(self):
```

Comment:
We can probably think of a better name for this method. Maybe something like "test_use_thank_you_sad" or something like that?